web/ui/dashboard/src/app/private/pages/project/event-logs/event-logs.component.ts

```diff
@@ -265,17 +265,20 @@ export class EventLogsComponent implements OnInit {
 		}
 	}
 
-	async retryEvent(requestDetails: { eventId: string }) {
+	async replayEvent(requestDetails: { eventId: string }) {
+		this.isRetrying = true;
 		try {
 			const response = await this.eventsLogService.retryEvent({ eventId: requestDetails.eventId });
 			this.generalService.showNotification({ message: response.message, style: 'success' });
+			this.isRetrying = false;
 			return;
 		} catch (error) {
+			this.isRetrying = true;
 			return error;
 		}
 	}
 
-	async batchRetryEvent() {
+	async batchReplayEvent() {
```

Comment:
Is there a reason you changed it to replay?
The action being performed is retry. It is the event in event log that is replayed